labs/lab-2-javascript-game/script-pt-3.js

```diff
@@ -0,0 +1,99 @@
+// debugger;
+
+var enemy;
+var user;
+var userHealthPercent;
+var enemyHealthPercent;
+
+window.addEventListener("load", () => {
+  document.getElementById("game-console").style.display = 'none';
+  document.getElementById("start-button").addEventListener("click", startGame);
+});
+
+class Character {
+  constructor(name, health, healsRemaining, winCount) {
+    this.name = name;
+    this.health = health;
+    this.healsRemaining = healsRemaining;
+    this.winCount = winCount;
+  }
+
+  generateAttackDamage() { // do min and max parameters / arguments
```

Comment:
I already gave you this note, and you put it in a comment -- but you could have one `randomInteger(min, max)` function that `attackDamage` and `healAmount` could both call.